### DISCUSSION TITLE
'Su closed' & 'Su off'

### DISCUSSION THREAD

**Discussion by merylstreet** [alltheplaces/alltheplaces#9502] [Ideas]
Maybe I missed the reason it is this way but I noticed a lot of opening_hours have flags 'closed' & 'off', specially for Sundays. 
Is there a reason why these are outputted (sometimes, not always)? 

From my understanding, in OSM they don't really add much information since they are assumed closed if hours are not defined.

Wouldn't it be better to drop them all to have consistent hours?

**Comment by CloCkWeRX** [comment]
I've added a small amount of documentation to https://github.com/alltheplaces/alltheplaces/pull/9505/files

**Reply by merylstreet** [reply]
Makes sense, thanks a lot for the quick clarification.
